config/routes.rb

```diff
@@ -41,6 +41,9 @@
       jsonapi_resources :plate_purposes, except: %i[update]
       jsonapi_resources :plate_templates
       jsonapi_resources :plates, except: %i[update]
+      jsonapi_resources :plates, only: [] do
```

Comment:
not sure you can have 2 `jsonapi_resources :plates` routes?

Reply:
Yes by adding `only: [] do` I’m not sure if this would reject any request, and only takes in the actions (get/ update etc)